### DISCUSSION TITLE
[Feature Request] Example on how to get code completion in MDX

### DISCUSSION THREAD

**Discussion by Knaackee** [microsoft/monaco-editor#3825] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

At first, thanks for this great project!

I already read https://github.com/microsoft/monaco-editor/issues/264 and https://github.com/microsoft/monaco-editor/pull/3096 plus googled a lot but didn't find an answer.

I am using the monaco editor to edit mdx (language=markdown). Syntax highlighting works as expected.

How to add code completion for html, javascript (typescript) and my own types of the React components I want to use?

I suggest a example or documention which i could provide if someone points me to the right direction.

**Comment by remcohaszing** [comment]
The reason there aren’t any examples for getting MDX intellisense in Monaco editor, is because it simply doesn’t exist.

I intend to start working on MDX intellisence myself soon (https://github.com/mdx-js/vscode-mdx/issues/165#issuecomment-1207387750) and have personal interest to support Monaco editor as a first-class citizen. (no ETA)

In the mean time you may be interested to have a look at [monaco-unified](https://monaco-unified.js.org)  to support linting. :)

**Reply by CapitaineToinon** [reply]
Hey, I see the PR above has been merged and mdx support is available for vscode. Is there any way from there to get it working with monaco now that the sitation has changed? Sorry for reviving this old issue but it comes up first in google results so I thought it would be the good place to ask for an update.
